src/simple_lottery/algorithm.rs

```diff
@@ -14,6 +14,7 @@ pub(super) fn prove(setup: &Setup, prover_set: &[Element]) -> Option<Proof> {
             element_sequence.push(element);
         }
         if prover_set.len() as u64 >= setup.proof_size {
+            element_sequence.sort();
```

Comment:
These elements are not necessarily unique

Reply:
I think we assume that `prover_set` must have distinct elements. Let's add this requirement to the function's doc? For now I added a debug assert.